examples/plot_simulate_alpha.py

```diff
@@ -29,32 +29,28 @@
 print(params)
 
 ###############################################################################
-# Now, we update a few parameters
+# Remove all evoked proximal feed parameters
+for key in params['*evprox*']:
```

Comment:
Much more understandable, but it ties in with the discussion on increasing transparency of the default params. As we think about developing tutorials for hnn-core, should the user make a habit of removing necessary keys? Or should they not exist in the "default" to begin with...

Reply:
These tutorials should be replaced to have drives instantiated through a clear API. I'm hoping it might be possible with #221 . cc @cjayb

Reply:
In this specific case, the issue of defining "default" will be (hopefully) resolved by the upcoming changes to the feed API where feeds will only be added to a parameter set if they are fully implemented in the simulation. The goal is to get rid of unnecessary parameters and objects that don't get fully instantiated during a full network build. As soon a we have a solid API for creating all types of feeds, it'll be super easy to redefine what "default" means.

Reply:
Sorry, I didn't see @jasmainak's [comment](https://github.com/jonescompneurolab/hnn-core/pull/213#discussion_r532934142) above until after writing my most recent post. I think we're on the same page though :)